at-quannguyen/W5/20180717105125_change_int_gender_to_users.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeIntGenderToUsers < ActiveRecord::Migration[5.1]
+  def up
+    User.where(gender: 'nam').update_all(gender: 0)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/7b01ff6d5f618841ca8c79e38047e5229d6d21f8/at-quannguyen/W5/20180717105125_change_int_gender_to_users.rb#L3"></span>Rails/SkipsModelValidations: Avoid using <code>update_all</code> because it skips validations. (http://guides.rubyonrails.org/active_record_validations.html#skipping-validations)</td>
    </tr>
  </tbody>
</table>